warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -407,36 +431,60 @@ def _get_browser_creation_function(self, browser_name):
         func_name = BROWSER_NAMES.get(browser_name.lower().replace(' ', ''))
         return getattr(self, func_name) if func_name else None
 
-
-    def _make_ff(self, webdriver_remote_url, desired_capabilites, profile_dir):
+    def _make_ff(self, webdriver_remote_url, desired_capabilites, profile_dir,
+                 binary, gecko_path):
         """Create an instance of firefox browser"""
-
-        if webdriver_remote_url:
-            browser = self._create_remote_web_driver(webdriver.DesiredCapabilities.FIREFOX,
-                                                     webdriver_remote_url, desired_capabilites,
-                                                     profile_dir)
-        else:
-            browser = webdriver.Firefox(firefox_profile=profile_dir)
-        return browser
-
-    def _make_chrome(self, webdriver_remote_url, desired_capabilities, profile_dir):
+        try:
+            if webdriver_remote_url:
+                browser = \
```

Comment:
no need of backslash as the next line can also be put here without affecting readability or pylint scores.